stdlib/private/SwiftPrivateThreadExtras/SwiftPrivateThreadExtras.swift

```diff
@@ -17,7 +17,7 @@
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
+#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(Android) || os(Cygwin) || os(Haiku)
```

Comment:
You've accidentally replaced `os(PS4)` with `os(OpenBSD)` instead of adding the latter.

```suggestion
#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
```

Reply:
Sorry about that. Accepted fix.

Reply:
No worries! Thank you for your effort.